scripts/chocolatey/build/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,230 @@
+<#
+.SYNOPSIS
+Installs the Lacework CLI.
+
+Authors: Technology Alliances <tech-ally@lacework.net>
+
+.DESCRIPTION
+This script installs the Lacework Command Line Interface.
+
+#>
+
+$ErrorActionPreference = "stop"
+
+Set-Variable GithubReleasesRootUrl -Option ReadOnly -value "https://github.com/lacework/go-sdk/releases" -Force
+Set-Variable PackageName -Option ReadOnly -value "lacework-cli-windows-amd64.exe.zip" -Force
+
+Function Get-File($url, $dst)
+{
+    Write-Host "Downloading $url"
+    try
+    {
+        [System.Net.ServicePointManager]::SecurityProtocol = [Enum]::ToObject([System.Net.SecurityProtocolType],3072)
+    }
+    catch
+    {
+        Write-Error "TLS 1.2 is not supported on this operating system. Upgrade or patch your Windows installation."
+    }
+    $wc = New-Object System.Net.WebClient
+    $wc.DownloadFile($url, $dst)
+}
+
+Function Get-WorkDir
+{
+    $parent = [System.IO.Path]::GetTempPath()
+    [string]$name = [System.Guid]::NewGuid()
+    New-Item -ItemType Directory -Path (Join-Path $parent $name)
+}
+
+Function Get-Archive()
+{
+    $url = $GithubReleasesRootUrl
+    $package_name = $PackageName
+
+    $lacework_cli_url = "$url/download/v${lacework_cli_version}/${package_name}"
+
+    $sha_url = "$lacework_cli_url.sha256sum"
+    $cli_dest = (Join-Path ($workdir) "lacework-cli.zip")
+    $sha_dest = (Join-Path ($workdir) "lacework-cli.zip.shasum256")
+
+    Get-File $lacework_cli_url $cli_dest
+    $result = @{ "zip" = $cli_dest }
+
+    try
+    {
+        Get-File $sha_url $sha_dest
+        $result["shasum"] = (Get-Content $sha_dest).Split()[0]
+    }
+    catch
+    {
+        Write-Warning "No shasum exists for $version. Skipping validation."
```

Comment:
Wondering if we should throw an error here? Is there a valid scenario where the CLI wouldn't have a shasum?

Reply:
That's a good point, let me raise a ticket to check this on the existing installer as well